tests/schema_test_package/modulea/modela.py

```diff
@@ -0,0 +1,5 @@
+from pydantic import BaseModel
```

Comment:
please remove all this and put it into the tests.

If you really need some python modules to test use `tmpdir` and write the files required within the test.

Reply:
Done